mica/report/tests/test_write_report.py

```diff
@@ -2,9 +2,20 @@
 import tempfile
 import os
 import shutil
+import pytest
 
 from .. import report
 
+HAS_SYBASE_ACCESS = False
+try:
+    import Ska.DBI
+    with Ska.DBI.DBI(server='sqlsao', dbi='sybase', user='aca_ops', database='axafocat') as db:
```

Comment:
I think making the actual db connection makes the most sense in this case.  If the tests are run from Py3, from GRETA, or as a user who doesn't have read access on the database, it seems reasonable to just skip the report-making test.